apps/fastapi_app.py

```diff
@@ -4,13 +4,16 @@
 from fastapi import FastAPI
 from time import sleep
 from asyncio import sleep as async_sleep
+from vulnpy.fastapi import vulnerable_routes
 
 app = FastAPI()
+app.include_router(vulnerable_routes)
 
-if os.environ.get("VULNPY_USE_CONTRAST"):
-    from contrast.fastapi import ContrastMiddleware
-
-    app.middleware("http")(ContrastMiddleware(app))
+# TODO: PYT-1701
+# if os.environ.get("VULNPY_USE_CONTRAST"):
+#     from contrast.fastapi import ContrastMiddleware
+#
+#     app.middleware("http")(ContrastMiddleware(app))
 
 
 @app.get("/")
```

Comment:
this is not hugely important (and I'm not sure if we're relying on this behavior somewhere else) but all other vulnpy example apps redirect `/` to `/vulnpy`